pkgs/development/python-modules/iterfzf/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fzf, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "iterfzf";
+  version = "0.5.0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "dahlia";
+    repo = pname;
+    rev = version;
+    sha256 = "0nrfq1zj4pwlgyd8ra5wl7skdbzvnxdh7wwhyh83y7yax8sp1xvj";
+  };
+
+  propagatedBuildInputs = [
+    fzf
+  ];
```

this will propagate the fzf command to the environment which should be avoided. Patching should be used instead
```suggestion
  substituteInPlace iterfzf/__init__.py  \
    --replace "EXECUTABLE_NAME = 'fzf.exe' if sys.platform == 'win32' else 'fzf'" \
      "EXECUTABLE_NAME = 'fzf.exe' if sys.platform == 'win32' else '${fzf}/bin/fzf'
```